plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -122,6 +122,7 @@
     private boolean skipTargetCleanupOnRollback;
 
     private boolean bucketExecutionEnabled = true;
+    private int partitionExecutionMinPartitions;
```

Comment:
this is someting we might want to enable by default if benchmarks results are good. cc @raunaqmorarka @przemekak

Reply:
curious how you'd scale this by default...if you have too few partitions the reduction in parallelism will outweigh the improved locality.